Editing "waste disposal process", "recycling process" and "anthropogenic litter"

iwilkie opened the pull request:
Dear editors,

I have added German, Italian, Spanish and Mandarin interpretations to "waste disposal process", "recycling process" and "anthropogenic litter", as mentioned in issue #1117.

The Italian and Spanish interpretations of "anthropogenic litter" were purposefully added as broader synonyms as the words "basura" and "rifuiti" do not directly translate to litter, but rather rubbish/waste in general (e.g. as household waste).

I have not included these terms in the "envoPlastics" subset as they are not specific to plastics and can include other materials/objects as well, please let me know if you still think it's appropriate to include it in the subset.

Thank you!

raissameyer commented:
> I have not included these terms in the "envoPlastics" subset as they are not specific to plastics and can include other materials/objects as well, please let me know if you still think it's appropriate to include it in the subset.

Indeed, those terms are not *specific* to plastics. However, I would still add them to the subset, as they may be *relevant* when considering plastics in the marine environment. 

The motivation behind the subset is not to say that any term in that subset is only applicable to the topic of that subset, but to be able to export a selected group of ENVO terms. The subset allows us to easily export a slimmed down, but still logically sound version of the ontology - here, containing all the terms that may be relevant to marine plastics. That subset can then be shared with UNEP, who thus won't have to interact with all of ENVO, but only with the compact selection of those terms relevant to their tasks.

raissameyer reviewed (commented):
Many thanks @iwilkie, looking good. 

Regarding adding these terms to the `envoPlastics` subset, please see my comment [above](https://github.com/EnvironmentOntology/envo/pull/1119#issuecomment-832506337).

iwilkie commented:
@raissameyer thank you for the clarification on the subsets, I have now added "waste disposal process", "recycling process" and "anthropogenic litter" to the envoPlastics subset.

pbuttigieg commented on src/envo/envo-edit.owl:
```diff
@@ -25305,6 +25305,13 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01000740> ObjectSomeValuesFrom(<
 # Class: <http://purl.obolibrary.org/obo/ENVO_01000741> (recycling process)
 
 AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://stats.oecd.org/glossary/detail.asp?ID=2260") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01000741> "A process in which some input bearing a waste role is consumed to produce an output which bears a resource role.")
```
>A process in which some input bearing a waste role is consumed to produce an output which bears a resource role

I think we could make this more readable

Maybe:

> A process during which materials deemed as waste are separated and converted into a form which can be used as input by the same processes which generated them.

pbuttigieg commented on src/envo/envo-edit.owl:
```diff
@@ -38994,10 +39007,17 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03500004> <http://purl.obolibrar
 # Class: <http://purl.obolibrary.org/obo/ENVO_03500005> (anthropogenic litter)
 
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03500005> "An anthropogenic environmental material consisting of any object no longer wanted and discarded outside of approved trash receptacles.")
```
>An anthropogenic environmental material consisting of any object no longer wanted and discarded outside of approved trash receptacles.


Not a fan of "trash", and litter need not be anthropogenically sourced. Further, in this case, litter isn't really a subclass of environmental material, more a role realised by it. Thus I would add: 

Under BFO:role:
> litter role =def. A role which 1) inheres in a material entity 2) is realised when that entity is discarded outside waste management systems by humans or their technologies.
 
and then, under BFO:material entity:
> anthropogenic litter =def. a material entity which has been discarded outside waste management systems by humans or their technologies.

Axioms would follow, and need a "littering process" to also be defined in a similar way. 

I also wonder if we need some sort of size cut-off. Very large things that are discarded are typically considered "debris" but this is quite loosely defined too.

raissameyer replied:
> I also wonder if we need some sort of size cut-off. Very large things that are discarded are typically considered "debris" but this is quite loosely defined too.

For the time being, we could add a comment noting this - however, as there is not clearly defined cut-off, I wouldn't add it to the definition.

raissameyer replied:
> I also wonder if we need some sort of size cut-off. Very large things that are discarded are typically considered "debris" but this is quite loosely defined too.

For the time being, we could add a comment noting this - however, as there is not clearly defined cut-off, I wouldn't add it to the definition.

pbuttigieg reviewed (changes requested):
Thanks for the PR @iwilkie @meichen-liu !

The multi-lingual content is much appreciated - but we could take this opportunity to refine definitions and add neighbourhood semantics.

iwilkie commented:
@pbuttigieg thank you for the comments.

I have now moved "anthropogenic" litter so that it is now under "material entity" and adjusted its definition as [indicated](https://github.com/EnvironmentOntology/envo/pull/1119#discussion_r626541050) by you.

I have added the term "litter role" under "role", and changed the definition of "recycling process" as you had [suggested](https://github.com/EnvironmentOntology/envo/pull/1119#discussion_r626532482).

raissameyer reviewed (commented):
Hi @iwilkie. I think it all looks good. Regarding @pbuttigieg's [comment](https://github.com/EnvironmentOntology/envo/pull/1119/files#r626541050), you could add that in a comment if you like.

iwilkie commented:
@raissameyer I have added a comment to "anthropogenic litter" stating that: Very large objects that are discarded are typically considered "debris", but this term is loosely defined and there are no standard size classifications which can be used to distinguish debris from litter.

raissameyer reviewed (approved):
Thanks for adding the comment @iwilkie. I think it looks good!

pbuttigieg reviewed (approved):
@iwilkie seems in order, let's see what happens to the other PRs when we merge.
